src/shared/components/Seperate.vue

```diff
@@ -0,0 +1,18 @@
+<script>
+  export default {
+    name: 'Separate',
+    functional: true,
+    props: {
+      tag: { type: String, default: undefined, },
+      separator: { type: String, default: ', ', },
+    },
+    render(createElement, context) {
+      const out = []
+      for (const [i, el] of context.slots().default.entries()) {
+        if (i !== 0) out.push(context.props.separator)
+        out.push(el)
+      }
+      return out
+    },
+  }
+</script>
```

Comment:
You can just duplicate this code where needed. It's 1 additional line compared to the `<Seperate>` component. (Render function components are not optimizable or compatible with vue3 and should therefore be avoided)

```suggestion
<template v-for="(artist, index) in item.artists">
    <template v-if="index > 0">, </template>
    <router-link :key="artist.id" :to="{name: 'artist', params: { id: artist.id }}" class="text-muted">
        {{ artist.name }}
     </router-link>
</template>
```

Reply:
ok sure, updated :+1:

Reply:
update again: just made the comma text-muted instead of the artist name